http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -157,12 +173,12 @@ protected LightblueResponse callService(HttpRequestBase httpOperation) {
                     HttpEntity entity = httpResponse.getEntity();
                     jsonOut = EntityUtils.toString(entity);
                     LOGGER.debug("Response received from service" + jsonOut);
-                    return new LightblueResponse(jsonOut, mapper);
+                    return new LightblueResponse(jsonOut);
                 }
```

Comment:
What if a consumer of LightblueHttpClient wants to set a ObjectMapper, how would the LightblueResponse know about it?